fvm/src/executor/default.rs

```diff
@@ -120,7 +120,8 @@ where
             // This error is fatal because it should have already been accounted for inside
             // preflight_message.
             if let Err(e) = cm.charge_gas(inclusion_cost) {
-                return (Err(e), cm.finish().1);
+                let (_, machine) = cm.finish();
```

Comment:
Hm... I think we need some form of `abort()` method? I guess it's fine here, but it would be nice to have that in the future.